src/nvim/fold.c

```diff
@@ -241,6 +241,8 @@ hasFoldingWin (
     return FALSE;
   }
 
+  if (last > win->w_buffer->b_ml.ml_line_count)
```

Comment:
Personally I wouldn't, but I guess it's a matter of opinion. I think the rule of thumb is to keep the commits focused so that the history is as clean as possible and `git blame`/`:Gblame` is as useful as possible.